### DISCUSSION TITLE
unable to use floating ui vue with teleport floating element

### DISCUSSION THREAD

**Discussion by BilalShah3** [floating-ui/floating-ui#2683] [Q&A]
i am using floating ui vue package.

the example they are provided is works fine when page is not scroll if we scroll the page the floating element is not anchored with refrence element. i have a requirement where i use float element inside teleport. i dont know how to handle this. if anyone helps me.

example link: https://github.com/tailwindlabs/headlessui/blob/main/packages/playground-vue/src/components/menu/menu-with-floating-ui.vue

**Comment by atomiks** [comment]
It's missing `autoUpdate`: https://floating-ui.com/docs/vue#anchoring

**Comment by BilalShah3** [comment]
but auto update works on conditional element. i didnot have condition on elelement.

**Comment by Nabil-MEDIOUNI** [comment]
@BilalShah3 you might want to consider a custom solution:

example you want to be able the see the content of a dropdown-list inside a dialog without being cut:

`
<template>
 <div ref="comboboxRef">
  <pill>Open Dropdown</pill>
  <dropdown-list :teleport-target="teleportTarget">
    <list-item>item 1</list-item>
    <list-item>item 2</list-item>
  </dropdown-list>
 </div>
</template>

const teleportTarget = computed(() => {
  const target = comboboxRef.value?.closest?.('dialog');
  return target ? target.id || null : null;
});`
